docs/keywords.json

```diff
@@ -6069,5 +6069,16 @@
     ],
     "category":"Text",
     "release":"3.1.0"
+  }, 
+  "cursor-plus":{
+    "alias":[
+      "3.2.0",
```

Comment:
I could just make the release searchable in the docs site if that's why we've been adding the version as an alias...

Reply:
That would work too.